src/Mado/QueryBundle/Repositories/BaseRepository.php

```diff
@@ -466,24 +454,24 @@ protected function relationship($queryBuilder)
      */
     public function onDuplicateUpdate($insertFields, $updateFields)
```

Comment:
Have you ever used this method?

Reply:
No but I think that for now It's better to don't remove It